presto-main/src/main/java/com/facebook/presto/server/ResourceUtil.java

```diff
@@ -315,7 +191,7 @@ public static String urlEncode(String value)
         }
     }
 
-    public static String urlDecode(String value)
+    private static String urlDecode(String value)
```

Comment:
[Note]
Why hide method from Util class and copy it instead of accessing from outside?
Also it's kinda funny that it was public even though it wasn't used outside, but we make it private when we may actually use it from outside ;-) From that perspective it can be interpreted as unrelated change ;-)

If it doesn't fit this Util class I would consider moving that to some new and better place.